src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -817,6 +821,18 @@ $kCodeBlockDarkLineMatchBackgroundColor: rgba(255, 255, 255, 0.12); // we don't
     gap: var(--kui-space-40, $kui-space-40);
   }
 
+  @media (min-width: $kui-breakpoint-laptop) {
+    .code-block-secondary-actions {
+      opacity: 0;
+      transition: opacity $kongponentsTransitionDurTimingFunc,  border $kongponentsTransitionDurTimingFunc;
```

Comment:
```suggestion
      transition: opacity $kongponentsTransitionDurTimingFunc, border $kongponentsTransitionDurTimingFunc;
```

It seems that our Styelint config is still not strict enough 😅

Reply:
Fixed the spacing